### PR TITLE
Implementation of a simple contract testing framework

### PR DESCRIPTION
**Pull request by EanCuznaivy**

### CODE REVIEW

File: AElf.Contracts.Consensus.Tests/ConsensusContractTestAElfModule.cs [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
 namespace AElf.Contracts.Consensus.Tests
 {
     [DependsOn(
-        typeof(ContractTestAElfModule)
-    )]
+        typeof(RuntimeSetupAElfModule),
+        typeof(AbpAutofacModule),
+        typeof(SmartContractExecutionAElfModule),
+        typeof(CSharpRuntimeAElfModule2),
+        typeof(KernelAElfModule),
```
Comment (loning) [inline AElf.Contracts.Consensus.Tests/ConsensusContractTestAElfModule.cs]: make KernelAElfModule depends on AElf.Kernel.* modules and here only depend on Kernel module and other useful module

File: AElf.Contracts.Consensus.Tests/ContractTestHelper.cs [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+using Google.Protobuf;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Moq;
+using Volo.Abp;
+using Volo.Abp.DependencyInjection;
+
+namespace AElf.Contracts.Consensus.Tests
+{
+    public class ContractTestHelper : ITransientDependency
```
Comment (loning) [inline AElf.Contracts.Consensus.Tests/ContractTestHelper.cs]: ContractTestHelper => ContractTester 

Helper is usually for static stateless method

File: AElf.Contracts.Consensus.Tests/ConsensusContractTestAElfModule.cs (hunk 1 above)
Comment (EanCuznaivy) [reply AElf.Contracts.Consensus.Tests/ConsensusContractTestAElfModule.cs]: Now use ContractTestAElfModule to add the dependents of other useful module.

**Comment by loning** [comment]
@EanCuznaivy CI failed

**Pull request merged**